crates/base/src/rt_worker/worker.rs

```diff
@@ -124,31 +121,36 @@ impl Worker {
                     }
                 });
 
+                let end_time = get_thread_time()?;
+                let cpu_time_used =
+                    usize::try_from((end_time - start_time) / 1_000_000).unwrap_or(0);
```

Comment:
When might this unwrap to err so it equals 0 ? Wondering if this could be a vector attack somehow

Reply:
both start and times comes from syscall to get CPU times. It returns a int64, but we convert it to a usize as it cannot technically be less than a 0. Defaults to a 0 in case value cannot be converted to a valid uszie.